scripts/automation/release/check.py

```diff
@@ -35,13 +46,16 @@ def check_all_component_revisions(r_start, r_end):
                     "response means no changes.")
     parser.add_argument('--component', '-c',
                         help='Component name (e.g. azure-cli, azure-cli-vm, etc.). If not '
-                             'specified, all component changes are shown.')
+                             'specified and --git-revision-start doesn\'t start with the component '
+                             'name, all component changes are shown.')
```

Comment:
Why are there single quotes and double quotes all in the same file?!?!?  Ahhhhh!